ibis/backends/duckdb/tests/test_register.py

```diff
@@ -213,9 +211,3 @@ def test_s3_403_fallback(con, httpserver, monkeypatch):
     # pyarrow, which indicates the fallback worked
     with pytest.raises(pa.lib.ArrowInvalid):
         con.read_parquet(httpserver.url_for("/myfile"))
-
-
-def test_register_numpy_str(con):
```

Comment:
Can we keep this test and xfail it?

Reply:
yeah -- I kept getting segfaults but it's related to a separate test, I'll investigate that a bit more